src/systems/diffeqs/sensitivity_transform.jl

```diff
@@ -0,0 +1,20 @@
+function forward_sensitivity_transform(sys::ODESystem,p::AbstractVector{<:Expression})::ODESystem
+    iv  = sys.iv()
+    f = du_dt = [eq.rhs for eq ∈ sys.eqs]
+    u = [state(iv) for state ∈ states(sys)] 
+    sen_symbols = [Symbol(:d,state.op.name,:_d,par.op.name) for state ∈ u, par in p]
+    du_dp = [Variable(sen_symbol)(iv) for sen_symbol ∈ sen_symbols]
```

Comment:
```suggestion
    names = [Symbol(state.op.name,:_,par.op.name) for state ∈ u, par ∈ p]
    du_dp = [Variable(name)(iv) for name ∈ names]
```
This does not specify the parametric type of `Variable{Type}`. Is basing this on `vartype(state.op)` good enough? Does the type of `par` not matter?

Is it really needed to use this naming scheme at this level already? Can we not keep the sensitivity states as Differential types?
```
D_p = [Differential(par.op.name) for par ∈ p]
[D_par(state) for state ∈ u, D_par ∈ D_p]
```
I think this would be beneficial for higher order derivatives.

Reply:
I think you can just keep them as differentials. That should work.